app/js/lib/view_handler/element_slide.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+
+	/**
+	 * View Handler for showing and hiding elements if the update value matches a regular expression
+	 * @class
+	 */
+	ElementSlider = {
+		el: null,
+		showOnValue: null,
+
+		// internal fields
+		slideSpeed: 600,
+		isHidden: null,
```

Comment:
No, it should be null to always trigger the show/hide event when the store is initialized. This makes the view Handler more independent form the initial HTML/CSS state, which may also be different for different instances of the view handler. Take for example period and bank data fields. In the default form, period is hidden and bank data fields are shown. If we initialize with `false` instead of `null`, the creator of the form must remember to remove the "display:none" code from the bank data fields, otherwise the visual state and the intended state differ.

Any recommendations on how to make this intention more clear in the code?